site/spi/Makefile-bits

```diff
@@ -16,26 +16,18 @@ ifeq ($(SP_OS), rhel7)
 
     ## If not overridden, here is our preferred LLVM installation
     ## (may be changed as new versions are rolled out to the facility)
-    LLVM_DIRECTORY ?= /shots/spi/home/lib/arnold/rhel7/llvm_4.0_final/
+    LLVM_DIRECTORY ?= /shots/spi/home/lib/arnold/rhel7/llvm_4.0
```

Comment:
The danger with the link is that changing it will "live change" any older builds that point to those files.

Seems a bit risky. I think I prefer to keep things explicit.